src/vs/workbench/contrib/search/browser/search.contribution.ts

```diff
@@ -268,6 +281,16 @@ MenuRegistry.appendMenuItem(MenuId.SearchContext, {
 	order: 2
 });
 
+MenuRegistry.appendMenuItem(MenuId.SearchContext, {
+	group: 'search',
+	order: 3,
+	command: {
+		id: RESTRICT_SEARCH_TO_FOLDER_ID,
+		title: nls.localize('restrictResultsToFolder', "Restrict Search to Folder...")
```

Comment:
I'll have to see how I feel about this name, honestly I thought "Find in Folder" is fine - when you have two actions that do exactly the same thing, I would lean towards giving them the same name. But it's fine for now, I don't have a strong opinion.